Warning: Unused Variables in SpectraIDViewTab.cpp

``` shell
/home/USER/Development/OpenMs/src/openms_gui/source/VISUAL/SpectraIDViewTab.cpp:325:13: warning: unused variable ‘current_identification_index’ [-Wunused-variable]
  325 |         int current_identification_index = item_pepid->data(Qt::DisplayRole).toInt();
      |             ^~~~~~~~~~~~~~~~~~~~~~~~~~~~
/home/USER/Development/OpenMs/src/openms_gui/source/VISUAL/SpectraIDViewTab.cpp:326:13: warning: unused variable ‘current_peptide_hit_index’ [-Wunused-variable]
  326 |         int current_peptide_hit_index = table_widget_->item(row, Clmn::PEPHIT_NR)->data(Qt::DisplayRole).toInt();
      |             ^~~~~~~~~~~~~~~~~~~~~~~~~
```

--- a/src/openms_gui/source/VISUAL/SpectraIDViewTab.cpp
+++ b/src/openms_gui/source/VISUAL/SpectraIDViewTab.cpp
@@ -322,8 +322,6 @@ namespace OpenMS
 
       if (item_pepid)
       {
-        int current_identification_index = item_pepid->data(Qt::DisplayRole).toInt();
-        int current_peptide_hit_index = table_widget_->item(row, Clmn::PEPHIT_NR)->data(Qt::DisplayRole).toInt();
 
         //array to store object of start-end postions, sequence and mod data of peptides;
         QJsonArray peptides_data;

